scanpipe/pipes/purldb.py

```diff
@@ -117,3 +117,13 @@ def match_resource(sha1_list, timeout=None, api_url=PURLDB_API_URL):
     if response and response.get("count"):
         packages = response["results"]
         return packages
+
+
+def index_package(purl, timeout=None, api_url=PURLDB_API_URL):
+    """Add PURL to PurlDB for indexing."""
+    payload = {"purl": purl}
+    response = request_get(
+        url=f"{api_url}packages/get_package/", payload=payload, timeout=timeout
```

Comment:
Is there an endpoint where we could provide multiple (all?) purls per request?
It seems pretty un-efficient to submit those one by one.

Reply:
AFAIK we don't have an endpoint for indexing multiple PURLs.
Maybe @JonoYang can confirm on this more.

Reply:
@JonoYang could we have this possibility on the PurlDB side?

Reply:
> @keshav-space The `index_package` is not great. What about `purldb.submit_package`?

Done

Reply:
Entered as https://github.com/nexB/purldb/issues/139